processor/src/fast/mod.rs

```diff
@@ -144,6 +144,10 @@ pub struct FastProcessor {
     /// return. It is a stack since calls can be nested.
     call_stack: Vec<ExecutionContextInfo>,
 
+    /// Mapping of digests of external procedure digests with their corresponding node ID and
+    /// forest containing the procedure.
+    pub(crate) called_external_procedures: BTreeMap<Word, (MastNodeId, Arc<MastForest>)>,
```

Comment:
Does a type alias make sense?

Reply:
I personally can read this signature just fine, and so prefer it as is